### PR TITLE
Titan Embedding Connector Blueprint content referenced by users of OpenSearch 2.11 version #2517 

### PR DESCRIPTION
**Pull request by mateon01**
### Description
Users using OpenSearch version 2.11 can also connect the connector.
 
### Issues Resolved
[issues link](https://github.com/opensearch-project/ml-commons/issues/2492)
 
### Check List
- [x] New functionality includes testing.
  - [ ] All tests pass
- [o] New functionality has been documented.
  - [x] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: docs/remote_inference_blueprints/bedrock_connector_titan_embedding_blueprint.md [hunk 1]
```diff
@@ -89,6 +89,72 @@ POST /_plugins/_ml/connectors/_create
...
+    {
+      "action_type": "predict",
+      "method": "POST",
+      "url": "https://bedrock-runtime.us-east-1.amazonaws.com/model/amazon.titan-embed-text-v1/invoke",
+      "headers": {
+        "content-type": "application/json",
+        "x-amz-content-sha256": "required"
+      },
+      "request_body": "{ \"inputText\": \"${parameters.inputText}\" }",
+      "pre_process_function": """
```
Comment (ylwu-amzn) [inline docs/remote_inference_blueprints/bedrock_connector_titan_embedding_blueprint.md:120]: This format is not so friendly for someone who is using curl to create connector. How about change to one line string with escaping ?

File: docs/remote_inference_blueprints/bedrock_connector_titan_embedding_blueprint.md [hunk 2]
```diff
@@ -89,6 +89,72 @@ POST /_plugins/_ml/connectors/_create
 }
 ```
 
+If you are using AWS OpenSearch Service version 2.11, there are no built-in functions for pre_process_function and post_process_function.
```
Comment (ylwu-amzn) [inline docs/remote_inference_blueprints/bedrock_connector_titan_embedding_blueprint.md:92]: Clarify the build-in pre/post process functions for Bedrock Titan Embedding model added in 2.12

File: docs/remote_inference_blueprints/bedrock_connector_titan_embedding_blueprint.md (hunk 1 above)
Comment (mateon01) [reply docs/remote_inference_blueprints/bedrock_connector_titan_embedding_blueprint.md:120]: is goot idean. i will change

File: docs/remote_inference_blueprints/bedrock_connector_titan_embedding_blueprint.md (hunk 2 above)
Comment (mateon01) [reply docs/remote_inference_blueprints/bedrock_connector_titan_embedding_blueprint.md:92]: Okay, I think that phrase should go in the part about using built-in functions at the top.

File: docs/remote_inference_blueprints/bedrock_connector_titan_embedding_blueprint.md [hunk 3]
```diff
@@ -89,6 +89,42 @@ POST /_plugins/_ml/connectors/_create
...
+    "model": "amazon.titan-embed-text-v1"
+  },
+  "credential": {
+    "roleArn": "<PLEASE ADD YOUR AWS ROLE ARN HERE>"
+  },
+  "actions": [
+    {
+      "action_type": "predict",
+      "method": "POST",
+      "url": "https://bedrock-runtime.us-east-1.amazonaws.com/model/amazon.titan-embed-text-v1/invoke",
```
Comment (ylwu-amzn) [inline docs/remote_inference_blueprints/bedrock_connector_titan_embedding_blueprint.md:115]:
```suggestion
      "url": "https://bedrock-runtime.${parameters.region}.amazonaws.com/model/${parameters.model}/invoke",
```

**Review by ylwu-amzn — APPROVED** [review]

**Review by b4sjoo — APPROVED** [review]

**Pull request merged**